### ISSUE TITLE
DEPOT - AquisitionPreis funktioniert für einige Banken nicht + FIX

### ISSUE THREAD

**Issue by SegFaulty** [nemiah/phpFinTS#464]
`getDepot->getStatement()->getHoldings()->[]->getAcquisitionPrice() `

erwartet das Format:

`:70E::HOLD//1STK23,968293+EUR`

einige Banken (z.b. DKB) und die Beispiele unter https://www.hbci-zka.de/dokumente/spezifikation_deutsch/fintsv3/FinTS_3.0_Messages_Finanzdatenformate_2010-08-06_final_version.pdf (Seite 162) liefern aber ein solches Format:

```
:70E::HOLD//1STK+805+00028+LU+20240822+
22,5624+EUR+
3
4
```

ich habe das für mich so gefixed:
nemiah/php-fints/lib/Fhp/MT535/MT535.php Line 50:

```
old:
if (preg_match('/:70E::HOLD\/\/\d*STK2(\d*),(\d*)\+([A-Z]{3})/sm', $block, $iwn)) {

new:
if (preg_match('/:70E::HOLD\/\/1STK(?:\+.*)?2(\d+),(\d+)\+([A-Z]{3})/sm', $block, $iwn)) {

```

**Comment by SegFaulty** [comment]
Ja, diese regex ist backward compatible zur alten, ich nutze das aktiv auch für Banken mit dem vorher funktionierenden Format

**Comment by nemiah** [comment]
Super, danke für den PR und closed :sunglasses:
